.travis.yml

```diff
@@ -31,14 +31,10 @@ matrix:
 
 
 before_install:
-  #- if [ "$QT_BASE" = "54" ]; then sudo add-apt-repository ppa:beineri/opt-qt542-trusty -y; fi
-  - if [ "$QT_BASE" = "57" ]; then sudo add-apt-repository ppa:beineri/opt-qt571-trusty -y; fi
   - sudo apt-get update -qq
 
 install:
-  - sudo apt-get install build-essential libgl1-mesa-dev
-  #- if [ "$QT_BASE" = "54" ]; then sudo apt-get install -qq qt54base; source /opt/qt54/bin/qt54-env.sh; fi
-  - if [ "$QT_BASE" = "57" ]; then sudo apt-get install -qq qt57base; source /opt/qt57/bin/qt57-env.sh; fi
+  - sudo apt-get install build-essential libgl1-mesa-dev qtbase5-dev
```

Comment:
Oh, I see. Sorry, I did it quite a while ago. Looks that I needed the latest Qt.